cupy/lib/polynomial.pyx

```diff
@@ -123,13 +126,11 @@ cdef class poly1d:
             return poly1d(self.coeffs * other)
         raise NotImplementedError
 
-    # TODO(Dahlia-Chehata): implement using polyadd
     def __add__(self, other):
-        raise NotImplementedError
-
-    # TODO(Dahlia-Chehata): implement using polyadd
-    def __radd__(self, other):
-        raise NotImplementedError
+        if isinstance(self, numpy.generic):
+            # for the case: numpy scalar + poly1d
+            raise TypeError('Invalid inputs')
```

Comment:
Write a more clear message.

Reply:
How about 'Numpy scalar and poly1d addition is not supported' ?